features/step_definitions/logging.rb

```diff
@@ -1389,3 +1389,22 @@ def check_log(logs, errors, exceptions)
     end
   end
 end
+
+Given /^I get storageclass from cluster and store it in the#{OPT_SYM} clipboard$/ do | sc |
+  sc = 'sc' unless sc
+
+  def get_storage_class()
```

Comment:
@QiaolingTang I am not sure it's a common practice to have a function def inside a cucumber step.  Why did you make it a function?

Reply:
@pruan-rht I couldn't find a good way to get a storage class if there is not default storage class in the cluster, so I defined this function.

Reply:
@QiaolingTang you shouldn't need to define a method within a step like what you have done.  The general rule of thumb is to make a method only if it's going to be called multiple times.  Since you are only using it once, there is no need to define a method.

Reply:
OK, let me update the step.